logos/swirge/swirge.svg

```diff
@@ -0,0 +1,9 @@
SHT2yyqWT8VXd9Fq1f9jv6+vrj+7fv7+N9gzo//oJLa5T5BqRbMBuz7ZmCt/WU3TP8D4AALDJUHwDAACbRSKRGK3FtxWTFYU6sP9Jca7CJuh/RTFE0eYZxUd1LpOvsyp6gKjo9JdwK+eaV1999ZJtttlmjR7v8mRr339jjfv1e7k7lz+Qnt/tO7DMHT867YaVxd5MYV5ra+tEu306Stnfb11Lr1QcFiQ6shN1a0xygfZfGGQAAECPQ/ENAABsVolEYhctbL63zwWJjuy2PCtYfCTY+j+f0XmMFefQg+n3P0eL9sUnu2/0I77vn6LHujyXX31mvTtx7jLXmA3u5ixox5pSN210tdsunYoy8dBrdJH+L1OSyeQfo5T9nwdpcbHia4pCBUYbFXiO9v97uAkAAHoqim8AAKAoJBKJA7SwItyEIPHBHtI5zKHROnow/d5t9Nd8Rfsi1M/0+52sWKXH+0W5TrvvnUb347fWRFvvN6Ii5U4YlXb7DIy9mcJaxTme590Spez/ardZn6U4T1Ho/2WjPK3oOEf7c6IOAEAvUOgWDwAAgE1q/fr1j2ixm2KK4i3LFWDzX00PV9HT6fduoxo3nMvvmEQiYbcUW6Gqyw4dVukGb3AbaU2J507dssbdsnNtrIU3/Zszihvq6ur6tRXe9H/xFHZ79csKu800X+HN5rs7U7GD9v+5gsIbAAC9BCPfAABA0YlGCFlDBhsBZA0a2put85dzonX0Avp9222YNuKrfVHqf/V73k/xhh4fE+U67Y/Lmtx1C+pdqZdwhwyrdJOGV7nK+Jsp/HLJkiWThw4d2hBmg//bflpcq9g9SHRkt9neqLhST2HdTAEAQC9D8Q0AABStRCJRq8UFitMVNlxpqWIcRYreR7/rGVpcFW7lHNza2roylUpt2JThv7Iz3J/8p8EdOLTCDS6N92YPvR6fbG5unlheXv5mlLL/z3gt7DbqLwWJjuyfeI/iQu3/RpABAAC9EsU3AABQ9BKJxFZaXKH4s85dNrxFEb2Afsc22tFuy7QuuG1eVOzs+/7f9fhHw1Tx0GvxzWw2OzmVSj0Rpez/MVSLWYppikLdHP5XYc0Ungo3AQBAb0bxDQAAAEUhEc6L9tNwK+drTU1Nvy8tLX1Nj8d332gX6Py5XnGm53l3Rin7t1dpcbbCbomutlweVkw8T/v+KtwEAAB9AcU3AAAAFIVEImH3h/5LYY032ixWbOv7/k/0+CFhavPQebM1+7hW/46Lwkzwb7bODtYkpE4xwnJ52P/BHr9Vz5EJMgAAoM+g2ykAAACKwvr1630tNuxka7dxnmuNDDZX4Up/r7lv4cKFNRsU3g7Qwm4dvV2Rr/DWqLhMsa32/76CwhsAAH0QI98AAABQVBKJxK+1aN+owIpY43zfP1uPfTNMbRo6V/5bU1PTxIqKikVRyv59E7SwZgpWfMsnq7BbUi/R/rn9AABA38TINwAAABQb63zafpRYpeLbdXV11qRgbZiKl/6eBZlMZs9EIrFPW+FN6yMUNsrNRrsVKrw9qthd+0+l8AYAAAwj3wAAAFB0EomEdbU9JdwK2C2pu2Wz2Y95nhdbx1udG69QnKa/474oZf8Wa6BwruJbCisE5vOc4lzt+/twEwAAIETxDQAAAEUnkUhsocUrivadQx/RuevnFW/bKLQo1y30nM2KKz3PuzRK2b8hpcU0xSyFzT2Xj41uu0Rxp/a3200BAADeh9tOAQAAUHTWr1//nhbXhls5ByQSic/7vn98tP2h6e/xFT+ZO3du9QaFt4O1eFbxA0W+wpvd/jpTsZ32v4PCGwAAKISRbwAAAChKiUSiSgsb/TY8SITs9s5dfd9/So9b44ONpvPgxxoaGial0+mlUcr+zt21sKLffkGiI5uL7g7FTO1vBUIAAIAPxMg3AAAAFKX169c3aHFxuJWzk2JKa2vrRD2+Ud8ia7eXtP8uiURi/7bCm9ZHK36q1X8pChXefqOYoP1PUVB4AwAAncLINwAAABStRCKR1GKuwopubd5RjPN9/wE9/sUw9d/pvHep9jk5mUw+FKXs+ftrcb7iDEW55fJ4WmGdVh8LNwEAADqPkW8AAAAoWuvDudSmh1s5dhvqt+rr64+OHv9A+jPrfN8/J5FIDGkrvGm9VGEFt1cV1sk0X+HtLcUUxR4U3gAAwMai+AYAAICitn79+v/R4pFwK2f6gAEDKrS8NdzsyApzilvnzJljzRRmWy4ROkyr8xXfVQyy/AbqFTYazpop/EThB1kAAICNQPENAAAAPYGNTmtfBKtWzKqrq/v6+vXrG8PU/1Hu0TVr1gxOJBKnHHHEEcHoOK3vrcUTijmKbS23gVbF/1OM1f5XKZqCLAAAwIfAnG8AAADoERKJhHUZPSHcCljn0QnZbPYAz/Out4TObedZM4bS0lLrkhrQfttocYXiSNu0XB6/UJyv/XP7AQAAdAeKbwAAAOgREonECC2sOFYZJEK/0fnslxW/833/+mQymbs9VX9+oBYXKk5XlFkuj38qpmv/v4SbAAAA3YviGwAAAHqMRCLxbS0uCrdyPqNz2j9F6/ZnrND2DcUFigGWy+N1hT1+n/blhBgAAMSG4hsAAAB6jEQiYXO9WYfSoUEi9LRiD4Wd2B6tuEyxlSKfFYrLFTfpPLg5yAAAAMSI4hsAAAB6lEQicaoWPwi3cq5W7KfYM9jqyAptNyku1/mvFeAAAAA2CYpvAAAA6FESiURKi2cV44PEB7OT3fsUF+i81241BQAA2KS8aAkAAAD0COvXr7cupzPCrQ9kTRT21p8/msIbAADYXBj5BgAAgB4pkUg8psWnw633sY6o5+s89xfhJgAAwOZD8Q0AAAA9UiKR+KgWT9pqkHBumeJSxS06x20NMgAAAJsZxTcAAAD0WIlE4qdaHK64QXG1zm3rLQ8AAFAsKL4BAACgx0okEsO0SOmc9j9hBgAAoJg49/8BX+3DQfOWqdgAAAAASUVORK5CYII="/>
```

Comment:
This is not an SVG, but an image wrapped in an SVG. Kindly upload an actual SVG file, preferably directly from the brand